teacher.rb

```diff
@@ -10,6 +10,5 @@ def can_use_services?
     true
   end
 end
-
-teacher = Teacher.new(29, 'PE', 'Samuel', parent_permission: false)
-puts teacher.can_use_services?
+# teacher = Teacher.new(29, 'PE', 'Samuel', parent_permission: false)
```

Comment:
### [OPTIONAL]
- [ ] It is always good practice to remove commented code. This will ensure your codebase is clean and professional